src/System.Text.Utf8/src/System/Text/Utf8/Utf8String.cs

```diff
@@ -606,5 +645,85 @@ private static byte[] GetUtf8BytesFromString(string s)
 
             return bytes;
         }
+
+        public Utf8String TrimStart()
+        {
+            CodePointEnumerator it = GetCodePointEnumerator();
+            while (it.MoveNext() && UnicodeCodePoint.IsWhitespace(it.Current))
+            {
+            }
+
+            return Substring(it.PositionInCodeUnits);
+        }
+
+        public Utf8String TrimStart(UnicodeCodePoint[] trimCodePoints)
```

Comment:
why add these versions that are not implemented? Why not just leave them out and add them entirely when they are actually coded?